client/my-sites/plan-features/header.jsx

```diff
@@ -210,28 +211,43 @@ export class PlanFeaturesHeader extends Component {
 	getPerMonthDescription() {
 		const {
 			discountPrice,
+			fullRawPrice,
 			rawPrice,
+			currencyCode,
 			translate,
 			planType,
 			currentSitePlan,
 			annualPricePerMonth,
 			isInSignup,
 			isMonthlyPlan,
+			isBillingTimeframeVariation,
 		} = this.props;
 
+		const isAnnualWpcomPlan = planMatches( planType, { group: GROUP_WPCOM, term: TERM_ANNUALLY } );
 		if ( isInSignup && isMonthlyPlan && annualPricePerMonth < rawPrice ) {
 			const discountRate = Math.round( ( 100 * ( rawPrice - annualPricePerMonth ) ) / rawPrice );
 			return translate( `Save %(discountRate)s%% by paying annually`, { args: { discountRate } } );
 		}
 
+		const fullPrice = formatCurrency( fullRawPrice, currencyCode, { stripZeros: true } );
 		if ( isInSignup && ! isMonthlyPlan ) {
+			if ( isAnnualWpcomPlan && isBillingTimeframeVariation ) {
+				return translate( '%(fullPrice)s billed annually', {
+					args: { fullPrice },
+				} );
+			}
 			return translate( 'billed annually' );
 		}
 
 		if ( typeof discountPrice !== 'number' || typeof rawPrice !== 'number' ) {
+			if ( isAnnualWpcomPlan && isBillingTimeframeVariation ) {
+				return translate( 'Per month, %(fullPrice)s billed yearly', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '%1$s per month, billed yearly', { context: 'Describes the amount of money per month, e.g. "$10 per month, billed yearly"'} )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/billing-timeframe-experiment-annual-plans)